service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -119,10 +121,11 @@ public OrderTypeAndDocument buildOrderTypeAndDocument(OrderTypeAndDocument typeA
         }
 
         orderBuilder.expiryDate(expiryDate)
+            .dateOfIssue(DateFormatterService.formatLocalDateToString(dateOfIssue, "d MMMM yyyy"))
             .type(typeAndDocument.getFullType(typeAndDocument.getSubtype()))
             .document(typeAndDocument.getDocument())
             .judgeAndLegalAdvisor(judgeAndLegalAdvisor)
-            .date(dateFormatterService.formatLocalDateTimeBaseUsingFormat(time.now(), "h:mma, d MMMM yyyy"));
+            .date(DateFormatterService.formatLocalDateTimeBaseUsingFormat(time.now(), "h:mma, d MMMM yyyy"));
```

Comment:
Could you make the strings that you are using constants in the `DateFormatterService`, there may already be the constants you require in there. Probably shouldn't change the ones in the test though as you want it to be in a specific format and if the constant is changed then the test should fail.

Reply:
Have added new constant for just DATE and used one of the existing ones